src/Command/RecipesCommand.php

```diff
@@ -96,35 +98,49 @@ protected function execute(InputInterface $input, OutputInterface $output)
             return 0;
         }
 
-        // display a resume of all packages
-        $write = [
-            '',
-            '<bg=blue;fg=white>                      </>',
-            '<bg=blue;fg=white> Available recipes.   </>',
-            '<bg=blue;fg=white>                      </>',
-            '',
-        ];
+        $outdated = $input->getOption('outdated');
 
+        $write = [];
         /** @var Recipe $recipe */
         foreach ($recipes as $name => $recipe) {
             $lockRef = $this->symfonyLock->get($name)['recipe']['ref'] ?? null;
 
-            $additional = '';
+            $additional = null;
             if (null === $lockRef && null !== $recipe->getRef()) {
                 $additional = '<comment>(recipe not installed)</comment>';
             } elseif ($recipe->getRef() !== $lockRef) {
                 $additional = '<comment>(update available)</comment>';
             }
-            $write[] = sprintf(' * %s %s', $name, $additional);
+
+            if (!$outdated) {
+                $write[] = sprintf(' * %s %s', $name, $additional);
+
+                continue;
+            }
+
+            if (null !== $additional) {
+                $write[] = sprintf(' * %s %s', $name, $additional);
+            }
         }
 
-        $write[] = '';
-        $write[] = 'Run:';
-        $write[] = ' * <info>composer recipes vendor/package</info> to see details about a recipe.';
-        $write[] = ' * <info>composer recipes:install vendor/package --force -v</info> to update that recipe.';
-        $write[] = '';
+        // Nothing to display
+        if (empty($write)) {
+            return 0;
+        }
```

Comment:
We also need to return 1 if the outdated option was passed AND there are some outdated items. 

I would probably remove the return 0 here and only worry about the return 1 here:

If the outdated flag was passed AND $write is NOT empty, return 1.

Reply:
I’m also not a huge fan of leveraging $write to know if we have at least 1 outdated package. I’d rather have a specific Boolean for that (like $hasOutdatedRecipes) that we initialize to false before the loop above and set to true inside of an outdated is found. But this is subjective.

Reply:
The return here allows nothing to be displayed as presentation text.

I'm not sure I updated as you want, please tell me if it's not ok